src/api/payment-requests/payment-requests.md

```diff
@@ -795,6 +795,49 @@ Alternatively you can provide an external transaction Id and the Centrapay [Asse
 | 403    | {% break _ REQUEST_EXPIRED %} | The cancellation cannot be completed because the `expiresAt` of the request has passed. |
 | 403    | {% break _ REQUEST_PAID %}    | The cancellation cannot be completed because the request `status` is `paid`.            |
 
+<a name="void">
+### Void a Payment Request **EXPERIMENTAL**
+
+Voiding a payment request will cancel the request and trigger any refunds if necessary.
+
+{% reqspec %}
+  POST '/api/payment-requests/{paymentRequestId}/void'
+  auth 'jwt'
+  path_param 'paymentRequestId', 'MhocUmpxxmgdHjr7DgKoKw'
+{% endreqspec %}
+
+{% h4 Example response payload %}
+{% json %}
+{
+  "type": "voided",
+	"cancellationReason": "CANCELLED_BY_MERCHANT",
+  "value": {
+    "currency": "NZD",
+    "amount": "1000",
+  },
+  "assetType": "centrapay.nzd.main",
+  "paymentRequestId": "MhocUmpxxmgdHjr7DgKoKw",
```

Comment:
Can you please add the `shortCode` field here? `"shortCode": "CP-C7F-ZS5-032"`

Reply:
Sure thing 😁